go.mod

```diff
@@ -45,3 +45,5 @@ replace (
 	k8s.io/cloud-provider => k8s.io/cloud-provider v0.17.0
 	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.5.14
 )
+
+replace github.com/submariner-io/shipyard => github.com/skitt/shipyard v0.0.0-20210401160514-2d121be1c8a3
```

Comment:
This will be removed before merging, once https://github.com/submariner-io/shipyard/pull/511 is merged.